Client/src/main/java/io/github/jwdeveloper/tiktok/http/mappers/LiveDataMapper.java

```diff
@@ -64,6 +64,10 @@ public LiveData.Response map(String json) {
                 default -> LiveData.LiveStatus.HostNotFound;
             };
             response.setLiveStatus(statusValue);
+        } else if (data.has("prompts") && jsonObject.has("status_code") &&
+            data.get("prompts").getAsString().isEmpty() && jsonObject.get("status_code").isJsonPrimitive()) {
+            // 4003110 is age restriction code
```

Comment:
4003110 is Magic number, Please put it to some kind of Constant variable

Reply:
Converted to constant!